.github/workflows/scripts/generate-weekly-report.js

```diff
@@ -190,6 +190,26 @@ async function getIssuesData({octokit, context}) {
   return stats
 }
 
+function generateComponentsLookingForOwnersReportSection(lookingForOwners) {
+  let count = 0
+  let section = []
+
+  for (const [componetType, components] of Object.entries(lookingForOwners)) {
+    count += Object.keys(components).length
```

Comment:
Do you think the output formatting might be a bit more clean if all component types are under a top-level `Components` `<summary>` element? I'm thinking this would make it look more uniform with the other entries in the weekly report. So the dropdown would be expanded to then show each of the component types.

Open for discussion though, this isn't a requirement by any means.